.github/workflows/R-CMD-check.yaml

```diff
@@ -171,6 +172,9 @@ jobs:
         shell: Rscript {0}
         run: |
           pak::local_system_requirements(execute = TRUE)
+          pak::pkg_system_requirements("renv", execute = TRUE)
+          pak::pkg_system_requirements("rsconnect", execute = TRUE)
+          pak::pkg_system_requirements("pkgdown", execute = TRUE)
       # xquartz and cairo are needed for Cairo package.
       # harfbuzz and fribidi are needed for textshaping package.
```

Comment:
Let's comment out this block if no longer relevant

Reply:
renv and rsconnect are needed. I've removed pkgdown